plugin/versioning/versioning.cc

```diff
@@ -30,15 +30,15 @@ static int forced_versioning_init(void *p __attribute__ ((unused)))
 
   DBUG_ENTER("forced_versioning_init");
   vers_force= true;
```

Comment:
No, there is no race. Besides, there is no difference from simple `set global vers_force = 1;`.